MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -679,4 +743,13 @@ - (void)runStressTestWithType:(MSALStressTestType)type
     }
 }
 
+- (void)hideCustomeWebViewIfNeed
+{
+    if (self.webviewTypeSegmentControl.selectedSegmentIndex == 0 &&
+        self.customWebviewTypeSegmentControl.selectedSegmentIndex == TEST_EMBEDDED_WEBVIEW_CUSTOM)
+    {
+       [self.customWebview loadHTMLString:@"<html><head></head></html>" baseURL:nil];
```

Comment:
isn't this what is loading about:blank?

Reply:
These code is supposed to be called at the end of the flow (end of the 1st call if no ssoSeeding or end of the ssoSeeding call). I guess the old code purpose us to 'reset' that custom webview but not sure we still need it or not. 
I will verify it today and make a new PR if that's redundant code can be removed.